assets/components/lvms/topolvm-controller_deployment.yaml

```diff
@@ -9,7 +9,7 @@ spec:
   revisionHistoryLimit: 10
   selector:
     matchLabels:
-      app.kubernetes.io/name: topolvm-controller
+      app.kubernetes.io/name: topolvm_controller
```

Comment:
I don't see this file in:
```
update_lvms_manifests() {
    local src="${STAGING_DIR}/lvms/amd64"
    local dest="${REPOROOT}/assets/components/lvms"
    local manifests=(
          "topolvm-controller_rbac.authorization.k8s.io_v1_clusterrolebinding.yaml"
          "topolvm-controller_rbac.authorization.k8s.io_v1_clusterrole.yaml"
          "topolvm-controller_v1_serviceaccount.yaml"
          "topolvm-csi-provisioner_rbac.authorization.k8s.io_v1_clusterrole.yaml"
          "topolvm-csi-provisioner_rbac.authorization.k8s.io_v1_clusterrolebinding.yaml"
          "topolvm-csi-provisioner_rbac.authorization.k8s.io_v1_role.yaml"
          "topolvm-csi-provisioner_rbac.authorization.k8s.io_v1_rolebinding.yaml"
          "topolvm-csi-resizer_rbac.authorization.k8s.io_v1_clusterrole.yaml"
          "topolvm-csi-resizer_rbac.authorization.k8s.io_v1_clusterrolebinding.yaml"
          "topolvm-csi-resizer_rbac.authorization.k8s.io_v1_role.yaml"
          "topolvm-csi-resizer_rbac.authorization.k8s.io_v1_rolebinding.yaml"
          "topolvm-metrics_rbac.authorization.k8s.io_v1_role.yaml"
          "topolvm-metrics_rbac.authorization.k8s.io_v1_rolebinding.yaml"
          "topolvm-node-metrics_v1_service.yaml"
          "topolvm-node_rbac.authorization.k8s.io_v1_clusterrole.yaml"
          "topolvm-node_rbac.authorization.k8s.io_v1_clusterrolebinding.yaml"
          "topolvm-node_v1_serviceaccount.yaml"
          "topolvm.io_logicalvolumes.yaml"
    )
```
How that line changed? Shouldn't we have this file in the `manifests` array?

Reply:
unfortunately this file doesn't exist as part of the manifest bundle.  This deployment (and the daemonset) are both programmatically defined in the LVMS operator code.  We have a ticket open with LVMS team to add a render type behavior for these.